File: cpp/level1_sum.cpp
//
//  level1_sum.cpp
//  programmers_training
//
//  Created by Chanwoo Noh on 2018. 06. 20..
//  Copyright © 2018년 Chanwoo Noh. All rights reserved.
//

#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    if (a == b)
        return a;
    if (a > b)
        std::swap(a,b);
    while (a <= b)
        answer += a++;
    return answer;
}

File: python3/level1_sum.py
#
#  level1_sum.py
#  programmers_training
#
#  Created by Chanwoo Noh on 2018. 09. 09..
#  Copyright © 2018년 Chanwoo Noh. All rights reserved.
#

def solution(a, b):
    sum = 0
    if a > b :
        a,b=b,a
    while a <= b :
        sum += a
        a+=1
    return sum

# ... or more simply
# def solution(a, b):
#     answer = 0
#     if a > b:   (a,b) = (b,a)
#     return sum(range(a,b+1))